packages/backend/src/models/CatalogModel/CatalogModel.ts

```diff
@@ -559,9 +557,7 @@ export class CatalogModel {
             );
         }
 
-        catalogItemsQuery = catalogItemsQuery
-            .orderBy('search_rank', 'desc')
-            .limit(limit ?? 50);
```

Comment:
`limit` was never defined based on usage, but it was always being applied by default to `50` - do we need to bring it back?

Reply:
I've added default 50 to the pagination args below. I think it's redundant to have both limit and pagination. Most likely, pagination was added later on to this model method.